app/authenticated/projects/edit/route.js

```diff
@@ -0,0 +1,16 @@
+import { hash } from 'rsvp';
+import { inject as service } from '@ember/service';
+import Route from '@ember/routing/route';
+
+export default Route.extend({
+    authzStore: service('authz-store'),
+    model: function (params) {
+        return hash({
+            project: this.get('authzStore').find('project', params.project_id),
+            projectRoleTemplateBindings: this.get('authzStore').findAll('projectRoleTemplateBinding', { forceReload: true })
```

Comment:
If you update to what I just pushed and update dependencies (ember-api-store), there's a `hasMany()` in `ember-api-store/utils/denormalize` so you can go to the project model and add

```
import { hasMany } from 'ember-api-store/utils/denormalize';

... Resource.extend({
...
  roleTemplateBindings: hasMany('id', 'projectRoleTemplateBinding', 'projectId'),
...
```

and then each `project` resource will have a live array of the associated `projectRoleTemplateBinding` resources.  You will need to preload them all (in `app/authenticated/route.js`, look for `this.preload(`)